.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,134 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: "Ubuntu Benchmark"
+
+on:
+  pull_request:
+    paths:
+      - 'velox/**'
+      - 'third_party/**'
+      - '.github/workflows/benchmark.yml'
+  push:
+    branches: [main]
+
+permissions:
+  contents: read
+
+defaults:
+  run:
+    shell: bash
+#TODO  concurrency groups?
+jobs:
+  benchmark:
+    runs-on: 8-core
+    container: ghcr.io/facebookincubator/velox-dev:amd64-ubuntu-22.04-avx
+    env:
+      CCACHE_DIR: "${{ github.workspace }}/.ccache/"
+      CCACHE_BASEDIR: "${{ github.workspace }}"
+      BINARY_DIR: "${{ github.workspace }}/benchmarks/"
+      LINUX_DISTRO: "ubuntu"
+      RESULTS_ROOT: "${{ github.workspace }}/benchmark-results"
+      BASELINE_OUTPUT_PATH: "${{ github.workspace }}/benchmark-results/baseline/"
+      TARGET_OUTPUT_PATH: "${{ github.workspace }}/benchmark-results/target/"
```

Comment:
`target` here is the same thing as `contender` in conbench parlance, correct? That is, this is the commit with code that is under test | under benchmark (i.e. the head of the PR branch on PR pushes or the new merge to main on merge-commits).

If so, could we use that same terminology here to prevent any confusion?

Reply:
Yes, I was thinking the same but used the existing wording. Will update.

Reply:
+1. I was planning to change that as well but never got the time.